rules/linux/persistence_udev_rule_creation.toml

```diff
@@ -13,10 +13,10 @@ malicious udev rules that trigger on specific events, executing arbitrary comman
 is plugged in or recognized by the system.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "endgame-*"]
-language = "kuery"
+index = ["logs-endpoint.events.file*"]
```

Comment:
Are we dropping endgame support because this is untested or because it really can not support endgame events?

Reply:
Because it uses the new `.Ext` field that is not supported by Endgame (at least in my stack the field is not present), and it is needed to reduce FPs by a lot.